--- sim-0.9.3/admin/cvs.sh~	2005-05-14 15:04:13.000000000 +0200
+++ sim-0.9.3/admin/cvs.sh	2005-05-14 15:04:13.000000000 +0200
@@ -65,7 +65,7 @@
     echo "*** SIM requires automake 1.5 or higher"
     exit 1
     ;;
-  automake*1.5* | automake*1.5-* | automake*1.6* | automake*1.7* | automake*1.8* ) : ;;
+  automake*1.5* | automake*1.5-* | automake*1.6* | automake*1.7* | automake*1.8 | automake*1.9* ) : ;;
   "" )
     echo "*** AUTOMAKE NOT FOUND!."
     echo "*** SIM requires automake 1.5 or higher"
